src/dynamics/thermal/node.cpp

```diff
@@ -57,17 +56,35 @@ void Node::PrintParam(void) {
   cout << "  node type    : " << node_type_str << endl;
   cout << "  heater id    : " << heater_id_ << endl;
   cout << "  Normal Vector: ";
-  for (int i = 0; i < 3; i++) {
+  for (size_t i = 0; i < 3; i++) {
     cout << normal_vector_b_[i] << " ";
   }
   cout << endl;
 }
 
 void Node::AssertNodeParams(void) {
-  assert(node_id_ >= 0);                                      // Node ID should be larger than 0
-  assert(heater_id_ >= 0);                                    // Heater ID should be larger than 0
-  assert(temperature_K_ >= environment::absolute_zero_degC);  // Temperature must be larger than zero kelvin
-  assert(capacity_J_K_ >= 0);                                 // Capacity must be larger than 0, use 0 when node is boundary or arithmetic
-  assert(0 <= alpha_ && alpha_ <= 1);                         // alpha must be between 0 and 1
-  assert(area_m2_ >= 0);                                      // Area must be larger than 0
+  // Temperature must be larger than zero kelvin
+  if (temperature_K_ < 0.0) {
+    std::cerr << "[WARNING] node: temperature is less than zero [K]." << std::endl;
+    std::cerr << "The value is set as 0.0." << std::endl;
+    temperature_K_ = 0.0;
+  }
+  // Capacity must be larger than 0, use 0 when node is boundary or arithmetic
+  if (capacity_J_K_ < 0.0) {
+    std::cerr << "[WARNING] node: capacity is less than zero [J/K]." << std::endl;
+    std::cerr << "The value is set as 0.0." << std::endl;
+    capacity_J_K_ = 0.0;
+  }
+  // alpha must be between 0 and 1
+  if (alpha_ < 0.0 && alpha_ > 1.0) {
```

Comment:
[MUST] ここはorじゃないとダメそうですね．．．
```suggestion
  if (alpha_ < 0.0 || alpha_ > 1.0) {
```